release-please-config.json

```
@@ -0,0 +1,160 @@
+{
+    "packages": {
+        ".": {
+            "last-release-sha": "da19158e7a02c574d4f6d3d367ee264cb08d47ec",
```

```suggestion
            "last-release-sha": "da19158e7a02c574d4f6d3d367ee264cb08d47ec",
            "include-component-in-tag": false,
```

I think we probably want to add `"include-component-in-tag": false,` to the config, to avoid the tag names becoming e.g. `node-gyp-v10.0.2` instead of `v10.0.2`?

---

Makes sense